app/src/components/shared/constants/surveyForm.constants.ts

```diff
@@ -0,0 +1,22 @@
+export const willingRadio = {

```

Comment:
I like what you're doing here - would these be better suited as `enum`s than constants?